src/test/kotlin/uk/gov/dluhc/notificationsapi/messaging/stubs/MessagingSupportLibraryListernerStub.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.dluhc.notificationsapi.messaging.stubs
+
+import io.awspring.cloud.sqs.annotation.SqsListener
+import org.slf4j.MDC
+import org.springframework.context.annotation.DependsOn
+import org.springframework.messaging.handler.annotation.Payload
+import org.springframework.stereotype.Component
+import uk.gov.dluhc.logging.config.CORRELATION_ID
+import uk.gov.dluhc.logging.config.MESSAGE_ID
+import uk.gov.dluhc.messagingsupport.MessageListener
+import java.util.Collections
+import java.util.UUID
+@Component
```

Comment:
it looks like some blank lines have got lost in the copy and paste in various places in this file